### PR TITLE
[cli] Fix `vc certs issue` subdomain parsing

### PR DESCRIPTION
**Pull request by onsclom**
I added a regression in https://github.com/vercel/vercel/commit/fd86f5d2c9fef8465a1514d6184c23c3bc1578dd#diff-4463dfbbb283602d090922c89d530aa8efb60baa260ffddcc33b275da6a3b00f where domains without a subdomain would incorrectly throw an error.

This fixes that and adds a test that catches this case.

### CODE REVIEW

File: packages/cli/src/commands/certs/issue.ts [hunk 1]
```diff
@@ -199,7 +199,7 @@ async function runStartOrder(
   const [header, ...rows] = dnsTable(
     pendingChallenges.map(challenge => {
       const subdomain = getSubdomain(challenge.domain);
-      if (!subdomain) {
+      if (subdomain === null) {
```
Comment (jeffsee55) [inline packages/cli/src/commands/certs/issue.ts:202]: What is the value here that led to the regression? `false` or `undefined`?

Comment (onsclom) [reply packages/cli/src/commands/certs/issue.ts:202]: Subdomain would be an empty string, and `!""` is `true`

Comment (onsclom) [reply packages/cli/src/commands/certs/issue.ts:202]: `getSubdomain` returns `null` for invalid domains and and empty string if the domain has no subdomain.

**Review by jeffsee55 — DISMISSED** [review]

File: packages/cli/test/unit/commands/certs/issue.test.ts [hunk 2]
```diff
@@ -30,6 +31,24 @@ describe('certs issue', () => {
     await expect(exitCodePromise).resolves.toEqual(1);
   });
 
+  it('should issue cert', async () => {
+    useUser();
+    client.scenario.post('v3/now/certs', (_, res) => {
```
Comment (TooTallNate) [inline packages/cli/test/unit/commands/certs/issue.test.ts]: Let's drop the `/now` part of the URL while we're in here? Will need to update where the fetch call is made as well.

**Review by TooTallNate — CHANGES_REQUESTED** [review]

File: packages/cli/test/unit/commands/certs/issue.test.ts [hunk 3]
```diff
@@ -30,6 +31,24 @@ describe('certs issue', () => {
     await expect(exitCodePromise).resolves.toEqual(1);
   });
 
+  it('should issue cert', async () => {
+    useUser();
+    client.scenario.post('v3/certs', (_, res) => {
```
Comment (TooTallNate) [inline packages/cli/test/unit/commands/certs/issue.test.ts]:
```suggestion
    client.scenario.post('/v3/certs', (_, res) => {
```

File: packages/cli/test/unit/commands/certs/issue.test.ts [hunk 4]
```diff
@@ -58,6 +77,26 @@ describe('certs issue', () => {
         },
       ]);
     });
+
+    it('should handle challenges', async () => {
+      useUser();
+      client.scenario.post('v3/certs', (_, res) => {
```
Comment (TooTallNate) [inline packages/cli/test/unit/commands/certs/issue.test.ts]:
```suggestion
      client.scenario.post('/v3/certs', (_, res) => {
```

**Review by TooTallNate — CHANGES_REQUESTED** [review]

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**